package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-network",
-  "version": "0.3.1",
+  "version": "0.3.2",
```

Comment:
@kellyselden is right, would you mind removing this from the PR? We will bump the version in the `package.json` when we publish to npm.